programs/mango-v4/src/state/group.rs

```diff
@@ -190,6 +194,17 @@ pub enum IxGate {
     TokenForceCloseBorrowsWithToken = 49,
     PerpForceClosePosition = 50,
     GroupWithdrawInsuranceFund = 51,
+    OpenbookV2CancelAllOrders = 52,
```

Comment:
Will need to update these numbers when rebasing

Reply:
Done